conf/application.test.conf

```diff
@@ -0,0 +1,22 @@
+# Root logger:
+logger.root=ERROR
+
+# Logger used by the framework:
+logger.play=ERROR
+
+# Logger provided to your application:
+logger.application=ERROR
+
+# ReactiveMongo
+mongodb.uri = "mongodb://localhost:27017/scalaplaymongo"
+
+# Only used in tests
+mongodb.dbName = "scalaplaymongo"
+
+# Override does not work, any ideas are welcome
+mongo-async-driver {
+  akka {
+    loglevel = ERROR

```

Comment:
I have no idea why, but this line is not overriding the application.config like the rest of the configuration.